modules/activiti-engine/src/main/java/org/activiti/engine/impl/persistence/entity/ThrowSignalEventSessionManager.java

```diff
@@ -0,0 +1,54 @@
+package org.activiti.engine.impl.persistence.entity;
+
+
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArraySet;
+import java.util.function.Function;

```

Comment:
This is a java 8 feature. 5.x supports java 1.6.

Reply:
Fixed. ThrowSignalEventSessionManager is removed in favor of CommandContext session attributed